### PR TITLE
fix():delete button removal

### LINKED ISSUE #4410: Change delete answer to remove entire element
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

Currently deleting an answer only removes the delete answer button:
deleteanswer

- [ ] 📝 **Update** this file here:
https://github.com/publiclab/plots2/blob/903cd43b996353226b4800ec21b06307e4066f3c/app/views/questions/_answer.html.erb#L133-L142

[See this page](https://publiclab.github.io/community-toolbox/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
        confirm: function () {
          $.ajax({
            url : "/answers/delete/<%= answer.id %>",
            type: 'GET',
            success: function(response){
              notyNotification('sunset', 3000, 'error', 'topRight', 'Answer deleted');
-               $("#a<%= answer.id %>delete-btn").remove()
+               $("#a<%= answer.id %>").remove()
            }
          });
        },
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by vnallamhawk**
Fixes #4410 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
Hi @vnallamhawk, welcome to Public Lab.
Please upload a screenshot/gif of the feature working.
Thanks for your PR

**Comment by vnallamhawk** [comment]
@oorjitchowdhary I am having issues running the project in my local environment. I am facing the following issues ```You don't have write permissions for the /Library/Ruby/Gems/2.3.0 directory.
Rakeshs-MBP:plots2 rakeshnallam$ bundle install --without production mysql``` in mac environment when I execute the command bundle install --without production mysql. Could someone help.

**Comment by oorjitchowdhary** [comment]
Try running `sudo bundle install` ..
Also, you can ask your installation doubts on #3840

**Comment by vnallamhawk** [comment]
Hello Team, I am facing multiple issues while setting up the codebase. Could you please test these changes and merge it. Thanks

File: OpenSource/.gitignore [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Autosave files
```
Comment (gauravano) [inline OpenSource/.gitignore]: Please remove this file

File: OpenSource/OpenSource.mdw [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<WorkspaceItem ctype="Workspace">
```
Comment (gauravano) [inline OpenSource/OpenSource.mdw]: Remove this file too.

File: app/views/questions/_answer.html.erb [hunk 3]
```diff
@@ -136,7 +136,7 @@
             type: 'GET',
             success: function(response){
               notyNotification('sunset', 3000, 'error', 'topRight', 'Answer deleted');
-              $("#a<%= answer.id %>delete-btn").remove()
+              $("#a<%= answer.id %>").remove()
```
Comment (gauravano) [inline app/views/questions/_answer.html.erb]: This seems good!

File: OpenSource/OpenSource.mdw (hunk 2 above)
Comment (vnallamhawk) [reply OpenSource/OpenSource.mdw]: okay

File: OpenSource/.gitignore (hunk 1 above)
Comment (vnallamhawk) [reply OpenSource/.gitignore]: okay

File: OpenSource/OpenSource.mdw (hunk 2 above)
Comment (vnallamhawk) [reply OpenSource/OpenSource.mdw]: @gauravano : i have updated the pull request . Thanks.

**Comment by gauravano** [comment]
Merged :tada: :balloon:

**Comment by gauravano** [comment]
Hi @vnallamhawk , thanks for nice work. We would love to have more help from you. If you are interested in solving more issues then check here - https://code.publiclab.org/#r=all for a new issue. In case, you are unable to find an issue, feel free to comment here. Thanks!

**Pull request merged**